context/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+// NOTE: This lint isn't applicable since we compile with nightly
+/* trunk-ignore(clippy/E0554) */
+#![feature(round_char_boundary)]
```

Comment:
This feature is for [`floor_char_boundary`](https://doc.rust-lang.org/std/primitive.str.html#method.floor_char_boundary) which is used to safely split strings on UTF-8 character boundaries